scripts/plugins/settings.py

```diff
@@ -39,6 +39,7 @@
 BITOPS_ENV_run_mode = os.environ.get("BITOPS_MODE")
 BITOPS_ENV_logging_level = os.environ.get("BITOPS_LOGGING_LEVEL")
 BITOPS_ENV_plugin_dir = os.environ.get("BITOPS_PLUGIN_DIR")
+BITOPS_ENV_installed_plugin_dir = os.environ.get("BITOPS_INSTALLED_PLUGIN_DIR")
```

Comment:
If we agreed to not expose the configuration for this setting via `bitops.config.yaml`, can we exclude setting it from the ENV var as a source too?

That's again for the reasons of simplicity and avoiding exposing configuration that's not needed by the user.

Reply:
Right but plugin scripts directly use this env var ...

Reply:
The exporting for the plugins looks fine.

But here we're importing the ENV var from the user's side. This could be removed as we don't want users to override this setting.

Reply:
This would make `BITOPS_INSTALLED_PLUGIN_DIR = /opt/bitops/scripts/installed_plugins/` a constant.

Reply:
Gotcha 👍  Will fix this after lunch